.husky/pre-push

```diff
@@ -21,7 +21,7 @@ fi
 
 # Run Unit Tests
 
-npm run test
+# npm run test
```

Comment:
Let’s remove the test and lint checks from the pre-push since it keeps getting in the way. These checks are already ran elsewhere in the pipeline.

Reply:
Removed